pages/common/twine.md

```diff
@@ -0,0 +1,21 @@
+# twine
+
+> Twine is a utility for publishing Python packages on PyPI.
+> Subcommands such as `twine upload` have their own usage documentation.
+> More information: <https://twine.readthedocs.io>.
+
+- Upload to the Test PyPI [r]epository and verify things look right:
+
+`twine upload -r testpypi dist/*`
+
+- Upload to PyPI:
+
+`twine upload dist/*`
+
+- Specify [u]sername and [p]assword while uploading to PyPI:
+
+`twine upload -u {{username}} -p {{password}} dist/*`
+
+- Upload to an alternative repository url:
```

Comment:
alternative like a mirror? or another registry?
```suggestion
- Upload to an alternative repository URL:
```

Reply:
Could be any valid repository (package index) URL. For some cases, you might only upload to a single url:
```
twine upload --repository-url <my pypi url> dist/*
```

Or perhaps issue twine upload multiple times to upload to different urls
```
twine upload --repository-url <my first pypi url> dist/*
twine upload --repository-url <my second pypi url> dist/*
```